### DISCUSSION TITLE
Enforce explicit return types on methods

### DISCUSSION THREAD

**Discussion by m1212e** [biomejs/biome#712] [Ideas]
In Nest.js there are a few features to auto generate docs/api specs based on typescript types. For this to work, the controllers (http endpoints in nest.js) must have methods which have an explicit return type.

In eslint, this would be the config:
```
  overrides: [
    {
      files: '*.controller.ts',
      rules: {
        '@typescript-eslint/explicit-function-return-type': 'error',
      },
    },
  ],
```
This enforces type annotations in controller classes. Would be very helpful to have this in biome! Thanks for your work!

**Comment by ematipico** [comment]
Thank you @m1212e for your proposal. Please use https://github.com/biomejs/biome/discussions/3 for proposing new rules :)

Also, someone else proposed it [here](https://github.com/biomejs/biome/discussions/3#discussioncomment-7313008). Feel free to add more value to the discussion

It would be great if someone could implement it :)

**Reply by m1212e** [reply]
Ahh, thanks, did not find it while searching. Will do!
